package.json

```diff
@@ -119,14 +120,15 @@
     "zx": "^1.14.1"
   },
   "peerDependencies": {
-    "@storybook/blocks": "^7.2.0",
-    "@storybook/client-logger": "^7.2.0",
-    "@storybook/components": "^7.2.0",
-    "@storybook/core-events": "^7.2.0",
-    "@storybook/manager-api": "^7.2.0",
-    "@storybook/preview-api": "^7.2.0",
-    "@storybook/theming": "^7.2.0",
-    "@storybook/types": "^7.2.0",
+    "@storybook/blocks": "^8.0.0",
+    "@storybook/client-logger": "^8.0.0",
+    "@storybook/components": "^8.0.0",
+    "@storybook/core-events": "^8.0.0",
+    "@storybook/manager-api": "^8.0.0",
+    "@storybook/preview-api": "^8.0.0",
+    "@storybook/theming": "^8.0.0",
+    "@storybook/types": "^8.0.0",
+    "@storybook/core-common": "^8.0.0",
```

Comment:
@tmeasday Before merging, let's remove these peer dependencies. We need to leave react (react & react-dom) ones for now. There seem to be issues otherwise, with installing the package.

Reply:
I ended up removing the react ones, and didn't notice any issues installing the package (I tried a SB7 and SB8 project): https://github.com/chromaui/addon-visual-tests/pull/167